sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-student-notification-second-disbursement-reminder.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    31,
+    'Student application notification for overdue second disbursement still pending.',
```

Comment:
Minor observation. Either `overdue of second disbursement confirmation` or `second disbursement still pending` alone sounds more clear IMO.